EncoreTickets.SDK.Tests/UnitTests/Authentication/JwtAuthenticationServiceTests.cs

```diff
@@ -1,154 +1,118 @@
-﻿using EncoreTickets.SDK.Api.Context;
+﻿using System.Collections.Generic;
+using System.Net;
+using EncoreTickets.SDK.Api.Context;
 using EncoreTickets.SDK.Api.Helpers;
-using EncoreTickets.SDK.Api.Results;
 using EncoreTickets.SDK.Api.Results.Exceptions;
-using EncoreTickets.SDK.Api.Results.Response;
 using EncoreTickets.SDK.Authentication;
 using EncoreTickets.SDK.Authentication.Models;
-using EncoreTickets.SDK.Utilities.Enums;
+using EncoreTickets.SDK.Tests.Helpers;
 using Moq;
 using NUnit.Framework;
+using RestSharp;
 
 namespace EncoreTickets.SDK.Tests.UnitTests.Authentication
 {
     internal class JwtAuthenticationServiceTests : JwtAuthenticationService
```

Comment:
Why is the test class inheriting from the tested class?

Reply:
We have to redefine the virtual Executor so as not to make real API calls